### ISSUE TITLE
[Add/Edit Observation] App crash when clicking "X"

### ISSUE THREAD

**Issue by gino-m** [google/ground-android#828]
```
    Process: com.google.android.gnd, PID: 25267
    java.lang.IllegalArgumentException: offset must be greater than or equal to 0
        at com.google.android.material.bottomsheet.BottomSheetBehavior.setExpandedOffset(BottomSheetBehavior.java:864)
        at com.google.android.gnd.ui.home.HomeScreenFragment.onGlobalLayout(HomeScreenFragment.java:240)
        at android.view.ViewTreeObserver.dispatchOnGlobalLayout(ViewTreeObserver.java:945)
        at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2250)
        at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1392)
        at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6752)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:911)
        at android.view.Choreographer.doCallbacks(Choreographer.java:723)
        at android.view.Choreographer.doFrame(Choreographer.java:658)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:897)
        at android.os.Handler.handleCallback(Handler.java:790)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Comment by gino-m** [comment]
Same when returning back from View Observation. For some reason in `BottomSheetMetrics`:
```
  /**
   * Calculates the expected height of the bottom sheet when fully expanded, assuming the sheet will
   * stop expanding just below the top toolbar.
   */
  public int getExpandedOffset() {
    return toolbarWrapper.getHeight() - marginTop;
  }
```

`marginTop > toolbarWrapper.getHeight()`

**Comment by gino-m** [comment]
This was fixed by #832.
